src/app/core/admin/feedback/admin-list-feedback.component.ts

```diff
@@ -35,31 +70,80 @@ export class AdminListFeedbackComponent extends AbstractPageableDataComponent<an
 			default: true
 		},
 		{ name: 'Type', sortable: true, sortField: 'type', sortDir: SortDirection.asc },
-		{ name: 'Feedback', sortable: false },
+		{ name: 'Feedback', sortable: false, sortField: 'body' },
+		{
+			name: 'Status',
+			sortable: true,
+			sortField: 'status',
+			sortDir: SortDirection.desc
+		},
+		{
+			name: 'Assignee',
+			sortable: true,
+			sortField: 'assignee',
+			sortDir: SortDirection.asc
+		},
+		{
+			name: 'Last Updated',
+			sortable: false,
+			sortField: 'updated',
+			sortDir: SortDirection.desc
+		},
 		{ name: 'Browser', sortable: true, sortField: 'browser', sortDir: SortDirection.asc },
 		{ name: 'OS', sortable: true, sortField: 'os', sortDir: SortDirection.asc },
 		{ name: 'Submitted From', sortable: true, sortField: 'url', sortDir: SortDirection.asc }
 	];
+	headersToShow: SortableTableHeader[] = [];
+
+	feedbackStatusOptions = FeedbackStatusOption;
+
+	assigneeUsernames: any;
 
 	constructor(
 		private feedbackService: FeedbackService,
 		private exportConfigService: ExportConfigService,
-		private alertService: SystemAlertService
+		private alertService: SystemAlertService,
+		private adminUsersService: AdminUsersService
 	) {
 		super();
+		this.adminUsersService
+			.getAll({ 'roles.admin': true }, 'username')
+			.pipe(take(1), untilDestroyed(this))
+			.subscribe({
+				next: usernames => {
+					this.assigneeUsernames = usernames;
+				}
+			});
 	}
 
 	ngOnInit() {
 		this.alertService.clearAllAlerts();
-
 		this.sortEvent$.next(this.headers.find((header: any) => header.default) as SortChange);
-
+		this.columnsUpdated(this.columns);
+		this.filterEvent$.next(this.getDefaultQuickFilters());
 		super.ngOnInit();
 	}
 
-	export() {
+	columnsUpdated(updatedColumns: any) {
```

Comment:
@sshah-asymmetrik can we please get some test coverage on these column changes and the selections for when a service call is made to export the feedback?